### DISCUSSION TITLE
OP CLIMB ISSUES

### DISCUSSION THREAD

**Discussion by WSpilot652** [flybywiresim/aircraft#3634] [Q&A] [answered]
Hey guys

Having issues with OP Climb on the MOD. Not sure if its pilot error or just a bug so i am starting here.

Every time i try to engage open climb it just selects V/S at +1500. It does this in all 3 versions. It also doesnt capture ALT* it just says V/S+0. I will attach screenshots as well. I have also had issues with ILS approaches where it captures LOC but not G/S. When G/S should be captured it just goes into V/S mode again. I may be doing something wrong here so appreciate your patience asi am not the most familiar with Airbus A/P.

Thank you in advance. This mod is amazing.

Brad Cherri
![Microsoft Flight Simulator 2021-02-19 1_08_57 PM](https://user-images.githubusercontent.com/79335605/108556888-ef55f600-72b4-11eb-958f-bdd0c3c9a5d1.png)
![Microsoft Flight Simulator 2021-02-19 1_09_06 PM](https://user-images.githubusercontent.com/79335605/108556904-f5e46d80-72b4-11eb-8986-e1cfca452c36.png)

**Comment by JJKHub** [comment]
I am not an expert or pilot, so i can not help you directly. But maybe this helps you to figure out if it is a bug or not, if you have the time:
https://youtube.com/playlist?list=PL4XNcpjBFH8ENxr_OnrSraok-fF69KWNi

The channel contains multiple videos explaining the a320 using the FBW custom build. There is also one for the A320 autopilot in general, ils approaches and one for the autoland feature.

**Comment by WSpilot652** [comment, marked as answer]
Thank you very much!

I think I may have figured it out. It had to do with my bravo throttle
quadrant. The autopilot on the quadrant was on V/s and I think that kept it
stuck in the sim. Still testing though.

Might be good if people could learn from this so feel free to share this
for others with the bravo throttle quadrant.

Brad

**Reply by JJKHub** [reply]
Mine is arriving in a couple of days, so good to know :)

**Comment by WSpilot652** [comment]
You will not be disappointed! It really adds to the experience!
